packages/tests-postman/data/service-providers.json

```diff
@@ -56,5 +56,22 @@
             "vcPrefix": "v1",
             "version": "pre"
         }
+    },
+    {
+        "name": "Transmute DID Actor API",
+        "serviceProvider": {
+            "provider": {
+                "name": "Transmute DID Actor API",
+                "url": "https://api.did.actor",
+                "contact": {
+                    "name": "Orie Steele",
+                    "email": "orie@transmute.industries"
+                },
+                "markets": [ "steel", "e-commerce" ]
+            },
+            "baseURL": "https://api.did.actor",
+            "vcPrefix": "next",
```

Comment:
I don't think this is right.... probably should be `api`... but I would have to look at the urls and match them up with what is published

Reply:
Pulled form the console logs... 

```
https://api.did.actor/api/credentials/issue
```